docs/tutorials/fundamentals/getting_started.md

```diff
@@ -89,14 +110,17 @@ is that you can preprocess your images and add multiple layers before displaying
 
 ### IPython console usage
 
-To launch napari from an IPython console import `napari` and create a `Viewer` object.
+To launch napari from an IPython console import `napari` and create a
+{class}`Viewer<napari.Viewer>` and {class}`Image<napari.layers.Image>` object.
 
 ```python
+# import sample data
+from skimage.data import cells3d
+
 import napari
-from skimage.data import astronaut
 
-# create the viewer and display the image
-viewer = napari.view_image(astronaut(), rgb=True)
+# create a `Viewer` and `Image` layer here
+viewer, image_layer = napari.imshow(cells3d())
 ```
 
 Napari will automatically use the interactive [`%gui qt` event
```

Comment:
Nitpick, butt he image below needs to be replaced.
It shows now only astronaut, but the ipython code is different, not using `imshow`
<img width="994" alt="image" src="https://user-images.githubusercontent.com/76622105/194818497-50dd77de-3c85-4e09-868d-cc46f700db7b.png">
(the other one is OK i think, because the console shows running a specific script.)

Reply:
Ah yes the images need updating, and I may as well do this for the other images here as well since they show an older version of napari (the menubar, toggles below the layerlist etc have since been updated). The only thing is that I run linux (menubar looks different, possibly other differences) while I notice that all (?) other images of viewer in the docs have been done with Mac. Should we be consistent? Thoughts? Maybe @jni wants to weigh in too?

Reply:
Happy to just fix the one related to `imshow` but the viewer itself looks different in linux vs mac - do we want to be consistent in the docs?

Reply:
For reference this is what napari looks like for me:
![Screenshot_2022-10-12_18-30-09](https://user-images.githubusercontent.com/23182829/195278354-58363e9d-2c3c-4b0b-a7a0-2a86d974a344.png)

It's really a nitpick but if all the other images in the docs are from mac, maybe this would look weird? Happy either way.